sbtplugin/src/main/scala/scala/scalanative/sbtplugin/ScalaNativePluginInternal.scala

```diff
@@ -59,18 +59,18 @@ object ScalaNativePluginInternal {
     nativeVerbose := false,
 
     nativeClang := {
-      val binaryName = "clang++"
+      val binaryNames = Seq("38", "37", "").map(ver => s"clang++$ver")
```

Comment:
This will also remove necessity of https://github.com/scala-native/scala-native/blob/master/docker/Dockerfile#L48 line in our docker file.